recipes/eukulele/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "1.0.2" %}
+{% set sha256 = "c21a85a8ea2e4a88ff9fc6828ab08e1ed6f05fbf45c94951948a219f0467891a" %}
+
+package:
+  name: eukulele
+  version: 1.0.2
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv"
+
+source:
+  url: https://github.com/AlexanderLabWHOI/EUKulele/archive/v1.0.2z.tar.gz
+  sha256: {{ sha256 }}
+
+test:
+  commands:
+    - EUKulele --version | grep '{{ version }}'
+    - EUKulele --help
+
+requirements:
+  build:
+    - python>=3.6
+#    - numpy
+    - biopython==1.77
+#    - pandas
+#    - seaborn
+    - chardet
+    - argparse
+#    - multiprocess
+#    - joblib
+#    - ujson
+#    - pyyaml
+#    - yaml
+    - pytest-runner   
+    - pytest-cov
+    - pytest-xdist 
+    - matplotlib-base
+ #   - blast 
+ #   - busco==4.1.4
+ #   - diamond 
+  run:
+    - python>=3.6
+    - numpy
+    - biopython
+    - pandas
+    - seaborn
+    - chardet
+    - argparse
+    - multiprocess
+    - joblib
+    - ujson
+    - pyyaml
+#    - yaml
```

Comment:
Please remove the commented out entries unless they need to be maintained to ease a future release.